Marlin/src/gcode/lcd/M0_M1.cpp

```diff
@@ -80,7 +80,10 @@ void GcodeSuite::M0_M1() {
 
   #elif ENABLED(EXTENSIBLE_UI)
 
-    ExtUI::onUserConfirmRequired(has_message ? args : MSG_USERWAIT); // SRAM string
+    if (has_message)
+      ExtUI::onUserConfirmRequired(args); // Can this take an SRAM string??
+    else
+      ExtUI::onUserConfirmRequired(GET_TEXT(MSG_USERWAIT));
```

Comment:
I'm still curious what we should do here. `onUserConfirmRequired` takes a progmem string, and this is the only place in the code that wants to use it differently.